docs/scripts/build_documentation.py

```diff
@@ -0,0 +1,89 @@
+"""Utility for building the documentation."""
+import argparse
+import os
+import pathlib
+from subprocess import check_call, run
+
+from utils import adjust_pictures
+
+from baybe.telemetry import VARNAME_TELEMETRY_ENABLED
+
+parser = argparse.ArgumentParser()
```

Comment:
Hi @AVHopp, you mention that you had troubles reusing the argument parser code. What exactly caused the problems?

Reply:
I wanted to create a function that initilaizes the `ArgumentParser` such that I can just import that function in the individual files and do not need to have all of the boilerplate code. However, this did not work as no matter where I added the arguments (in that function, after initializing the parser....), it did not properly recognize it :(

Reply:
I see no reason at the moment why it shouldn't work. Can you share what you tried? Perhaps we can debug together

Reply:
Redundant by now.